clustered/ehcache-client/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
   implementation "org.terracotta:lease-api:$terracottaPlatformVersion"
   implementation "org.terracotta.dynamic-config.entities:dynamic-config-topology-entity-client:$terracottaPlatformVersion"
   implementation "org.terracotta:connection-api:$terracottaApisVersion"
+  implementation group: 'org.terracotta', name: 'terracotta-utilities-tools', version: parent.terracottaUtilitiesVersion
```

Comment:
It will be nice to maintain consistency while adding dependencies using format 
`implemenation "org.terracotta:terracotta-utilities-tools:$terracottaUtilitiesVersion`